pkg/cmd/registry/artifact/crud/get/get.go

```diff
@@ -0,0 +1,165 @@
+For fetching artifacts using global identifiers please use "service-registry download" command
+`,
+		Example: `
+## Get latest artifact by name
+rhoas service-registry artifact get myschema
+
+## Get latest artifact and save its content to file
+rhoas service-registry artifact get myschema myschema.json
+
+## Get latest artifact and pipe it to other command 
+rhoas service-registry artifact get myschema | grep -i 'user'
+
+## Get latest artifact by specifying custom group, registry and name as flag
+rhoas service-registry artifact get --group mygroup --instance-id=myregistry --artifact myartifact
+`,
+		Args: cobra.RangeArgs(0, 2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				opts.artifact = args[0]
+			}
+
+			if opts.artifact == "" {
+				return errors.New("Artifact is required. Please specify artifact as positional argument or by using --artifact flag")
+			}
+
+			if len(args) > 1 {
+				opts.outputFile = args[1]
+			}
+
+			if opts.registryID != "" {
+				return runGet(opts)
+			}
+
+			cfg, err := opts.Config.Load()
+			if err != nil {
+				return err
+			}
+
+			if !cfg.HasServiceRegistry() {
+				return errors.New("No service Registry selected. Use 'rhoas service-registry use' to select your registry")
+			}
+
+			opts.registryID = fmt.Sprint(cfg.Services.ServiceRegistry.InstanceID)
+			return runGet(opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.artifact, "artifact", "a", "", "Id of the artifact")
+	cmd.Flags().StringVarP(&opts.group, "group", "g", "", "Group of the artifact")
+	cmd.Flags().StringVarP(&opts.registryID, "instance-id", "", "", "Id of the registry to be used. By default uses currently selected registry")
+	cmd.Flags().StringVarP(&opts.outputFile, "output-file", "", "", "Filename of the output file")
+	cmd.Flags().StringVarP(&opts.version, "version", "", "", "Version of the artifact")
+
+	flagutil.EnableOutputFlagCompletion(cmd)
+
+	return cmd
+}
+
+func runGet(opts *Options) error {
+	logger, err := opts.Logger()
+	if err != nil {
+		return err
+	}
+
+	conn, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	dataAPI, _, err := conn.API().ServiceRegistryInstance(opts.registryID)
+	if err != nil {
+		return err
+	}
+
+	if opts.group == "" {
+		logger.Info("Group was not specified. Using " + util.DefaultArtifactGroup + " artifacts group.")
+		opts.group = util.DefaultArtifactGroup
+	}
+
+	ctx := context.Background()
+	var dataFile *os.File
+	if opts.version != "" {
+		logger.Info("Fetching artifact with version: " + opts.version)
+		request := dataAPI.VersionsApi.GetArtifactVersion(ctx, opts.group, opts.artifact, opts.version)
+		dataFile, _, err = request.Execute()
+	} else {
+		logger.Info("Fetching latest artifact")
+		request := dataAPI.ArtifactsApi.GetLatestArtifact(ctx, opts.group, opts.artifact)
+		dataFile, _, err = request.Execute()
+	}
+	if err != nil {
+		return registryinstanceerror.TransformError(err)
```

Comment:
Sadly backend doesn't provide proper error codes thus we have limited ability to map anything. 
What we get right now is 

- message
- Name of the Exception class (that is unlikely to be stable)
- ugly java stacktrace (in dev env)

CC @jsenko do you think that it would be possible to introduce error codes to dataplane in the future?